src/framework/theme/components/select/select.component.ts

```diff
@@ -849,7 +862,7 @@ export class NbSelectComponent
       return this.selectionModel.map((option: NbOptionComponent) => option.content).join(', ');
     }
 
-    return this.selectionModel[0].content;
+    return this.selectionModel[0]?.content;
```

Comment:
why do we need this change?

Reply:
Earlier `selectionView` variable was used in the button only when something was selected. Now it is used in `optionSearchInput` all the time